src/Expander.hs

```diff
@@ -1072,30 +1083,9 @@ addRootScope' stx = do
   return (addScope' stx rsc)
 
 
-forkExpandOneDecl :: DeclPtr -> Scope -> DeclValidityPtr -> Syntax -> Expand ()
-forkExpandOneDecl dest sc phaseDest stx = do
-  forkExpanderTask $ ExpandDecl dest sc stx phaseDest
-
-forkExpandDecls :: ModBodyPtr -> Syntax -> Expand ()
-forkExpandDecls dest (Syntax (Stx _ _ (List []))) =
-  modifyState $ over expanderCompletedModBody (<> Map.singleton dest Done)
-forkExpandDecls dest (Syntax (Stx scs loc (List (d:ds)))) = do
-  -- Create a scope for this new declaration
-  sc <- freshScope $ T.pack $ "For declaration at " ++ shortShow (stxLoc d)
-  restDest <- liftIO $ newModBodyPtr
-  declDest <- liftIO $ newDeclPtr
-  declPhase <- newDeclValidityPtr
-  modifyState $ over expanderCompletedModBody $
-    (<> Map.singleton dest (Decl declDest restDest))
-  forkExpanderTask $
-    ExpandMoreDecls restDest sc
-      (Syntax (Stx scs loc (List ds)))
-      declPhase
-  forkExpandOneDecl declDest sc declPhase =<< addRootScope d
-  where
-    stxLoc (Syntax (Stx _ srcloc _)) = srcloc
-forkExpandDecls _dest _other =
-  error "TODO real error message - malformed module body"
```

Comment:
`forkExpandDecls` is now `expandDeclForm`, and has been moved next to `expandDeclForm` (formerly `expandOneDeclaration`)